testutil/integration/suites/interface.go

```diff
@@ -0,0 +1,45 @@
+//go:build integration
+
+package suites
+
+import (
+	"testing"
+
+	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+
+	"github.com/pokt-network/poktroll/testutil/integration"
+)
+
+// IntegrationSuite is an interface intended to be used within test suites which
+// exercise an integration.App.
+type IntegrationSuite interface {
```

Comment:
This is great!

Just note, you'll be responsible for reviewing tests now ;)

I have a general idea of what's available, but one only gets good at it once you do something hands on.